tests/CocktailParty.tex

```diff
@@ -9,46 +9,45 @@ \subsection{Focus}
 \subsection{Setup}
 \begin{itemize}
 	\item \textbf{Party room}: any (large) room inside the apartment when normally a party would be held.
-	\item \textbf{Guests:} At least five people are distributed in a predefined \quotes{party room} either sitting or standing, some of them forming groups of 2 or 3 people, from which at least one is sitting. Three of the guests have drink orders assigned by the referees. The sitting person always has an assigned drink.
-	\item \textbf{Bar:} The bar is any flat surface where objects can be placed, in a room other than the \quotes{party room}. All available beverages are on top of the bar for the robot can see them.
-	\item \textbf{Barman:} The Barman bay be standing either behind the bar or next to it, depending on the arena setup.
+	\item \textbf{Guests:} At least five people are distributed in a predefined \quotes{party room} either sitting or standing.
+                Three of the guests have drink orders.
+	\item \textbf{Bar:} The bar is any flat surface where objects can be placed, in a room other than the \quotes{party room}.
+                All available beverages are on top of the bar.
+	\item \textbf{Barman:} The Barman may be standing either behind the bar or next to it, depending on the arena setup.
 \end{itemize}
 
 \subsection{Task}
 
 \begin{enumerate}
 
-	\item \textbf{Entering:} The robot enters the arena and navigates to the party room and waits for being called.
-
+	\item \textbf{Entering:} The robot enters the arena and navigates to the party room.
 	\item \textbf{Getting called:} The guests call the robot simultaneously, either rising an arm, waving, or shouting. The robot has to approach one of them.
-	% Remark: Themself is the correct word used instead of himself/herself  when person gender is unknown.
-	The calling person introduces themself by name before giving the order of a drink. The robot leads the dialogue to learn the person and retrieve their drink order. \\
-
-	The robot can decide to skip the detection of the calling and ask one person to walk in front of it. In this case, the referees determine the person to approach the robot.
+        Optionally, the robot can skip the call detection and ask for a person to step in front of it (the referees determine who approaches to the robot).
 
-	\item \textbf{Taking the order:} After the robot has fetched the order of the first guest, it can either fetch more orders (i.e.~ find next calling guest or looking for the sitting one) or proceed to place the order. In the first case, the robot searches for the remaining calling people. During the search process, the robot is allowed to either ask people to call for it again, or to ask people to come to it and to give a new order. In both cases the robot may call into the room.
+	The calling person introduces themself by name before giving the order of a drink.
+	The robot leads the dialogue to learn the person and retrieve their drink order. \\
```

Comment:
The robot leads a dialogue to determine the person's name and obtain their drink order. \\